packages/@uppy/locales/src/el_GR.js

```diff
@@ -0,0 +1,152 @@
+  noInternetConnection: 'Δεν υπάρχει σύνδεση στο Internet',
+  openFolderNamed: 'Άνοιγμα φακέλου %{name}',
+  pause: 'Παύση',
+  pauseUpload: 'Παύση μεταφόρτωσης',
+  paused: 'Έγινε παύση',
+  poweredBy: 'Με τη δύναμη τού',
+  preparingUpload: 'Προετοιμασία μεταφόρτωσης...',
+  processingXFiles: {
+    '0': 'Προετοιμασία %{smart_count} αρχείων',
+    '1': 'Προετοιμασία %{smart_count} αρχείου',
+    '2': 'Προετοιμασία %{smart_count} αρχείων'
+  },
+  removeFile: 'Αφαίρεση αρχείου',
+  resetFilter: 'Επαναφορά φίλτρου',
+  resume: 'Συνέχεια',
+  resumeUpload: 'Συνέχεια μεταφόρτωσης',
+  retry: 'Προσπάθεια ξανά',
+  retryUpload: 'Προσπάθεια μεταφόρτωσης ξανά',
+  saveChanges: 'Αποθήκευση αλλαγών',
+  selectAllFilesFromFolderNamed: 'Επιλογή όλων των αρχείων από τον φάκελο %{name}',
+  selectFileNamed: 'Επιλογή αρχείου %{name}',
+  selectX: {
+    '0': 'Επιλογή %{smart_count}',
+    '1': 'Επιλογή %{smart_count}',
+    '2': 'Επιλογή %{smart_count}'
+  },
+  smile: 'Χαμογελάστε!',
+  startRecording: 'Ξεκίνημα εγγραφής βίντεο',
+  stopRecording: 'Σταμάτημα εγγραφής βίντεο',
+  takePicture: 'Βγάλτε μια φωτογραφία',
+  timedOut: 'Η μεταφόρτωση σταμάτησε για %{seconds} δευτερόλεπτα, γίνεται ακύρωση.',
+  unselectAllFilesFromFolderNamed: 'Αποεπιλογή όλων των αρχείων από τον φάκελο %{name}',
+  unselectFileNamed: 'Αποεπιλογή αρχείου %{name}',
+  upload: 'Μεταφόρτωση',
+  uploadComplete: 'Μεταφόρτωση ολοκληρώθηκε',
+  uploadFailed: 'Μεταφόρτωση απέτυχε',
+  uploadPaused: 'Μεταφόρτωση σε παύση',
+  uploadXFiles: {
+    '0': 'Μεταφόρτωση %{smart_count} αρχείων',
+    '1': 'Μεταφόρτωση %{smart_count} αρχείου',
+    '2': 'Μεταφόρτωση %{smart_count} αρχείων'
+  },
+  uploadXNewFiles: {
+    '0': 'Μεταφόρτωση +%{smart_count} αρχείων',
+    '1': 'Μεταφόρτωση +%{smart_count} αρχείου',
+    '2': 'Μεταφόρτωση +%{smart_count} αρχείων'
+  },
+  uploading: 'Γίνεται μεταφόρτωση',
+  uploadingXFiles: {
+    '0': 'Μεταφορτώνονται %{smart_count} αρχεία',
+    '1': 'Μεταφορτώνεται %{smart_count} αρχείο',
+    '2': 'Μεταφορτώνονται %{smart_count} αρχεία'
+  },
+  xFilesSelected: {
+    '0': '%{smart_count} επιλεγμένα αρχεία',
+    '1': '%{smart_count} επιλεγμένο αρχείο',
+    '2': '%{smart_count} επιλεγμένα αρχεία'
+  },
+  xMoreFilesAdded: {
+    '0': '%{smart_count} ακόμα αρχεία προστέθηκαν',
+    '1': '%{smart_count} ακόμα αρχείο προστέθηκε',
+    '2': '%{smart_count} ακόμα αρχεία προστέθηκαν'
+  },
+  xTimeLeft: '%{time} απομένουν',
+  youCanOnlyUploadFileTypes: 'Μπορείτε να ανεβάσετε μόνο: %{types}',
+  youCanOnlyUploadX: {
+    '0': 'Μπορείτε να ανεβάσετε μόνο %{smart_count} αρχεία',
+    '1': 'Μπορείτε να ανεβάσετε μόνο %{smart_count} αρχείο',
+    '2': 'Μπορείτε να ανεβάσετε μόνο %{smart_count} αρχεία'
+  },
+  youHaveToAtLeastSelectX: {
+    '0': 'Πρέπει να επιλέξετε τουλάχιστον %{smart_count} αρχεία',
+    '1': 'Πρέπει να επιλέξετε τουλάχιστον %{smart_count} αρχείο',
+    '2': 'Πρέπει να επιλέξετε τουλάχιστον %{smart_count} αρχεία'
+  }
+};
```

Comment:
```suggestion
}
```

Reply:
So the linter doesn’t complain — Uppy uses https://standardjs.com/rules.html, which has a “no semicolons, unless necessary” rule :)